app/assets/src/components/views/SampleView/SampleView.jsx

```diff
@@ -268,6 +288,17 @@ class SampleView extends React.Component {
     return {};
   };
 
+  // TODO (gdingle): refactor with onShareClick in SamplesHeatmapView?
+  onShareClick = () => {
```

Comment:
The onshare will use the new id to a visualization in the future, right?

Reply:
I'm not sure. I assumed no, because you wouldn't normally expect "share" to have any side effects.

Reply:
Not what I was thinking... but see is not very clear.
So, good for now and lets discuss with Jenn what is the behavior we want.